weblate/checks/fluent/utils.py

```diff
@@ -0,0 +1,255 @@
+# Copyright © Henry Wilkes <henry@torproject.org>
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import annotations
+
+import re
+from typing import TYPE_CHECKING
+
+from django.utils.html import escape, format_html, format_html_join
+from translate.storage.fluent import (
+    FluentPart,
+    FluentReference,
+    FluentSelectorBranch,
+    FluentUnit,
+)
```

Comment:
Fluent support is optional in Weblate, but this makes it a requirement. Solution is needed:

* Deal with the import failure here.
* Adjust dependencies to require fluent.syntax unconditionally.

Reply:
Which would you prefer?

I guess with this added support it could be considered unconditional.

Reply:
As it doesn't pull any huge third-party deps, I'd go for making it required.

Reply:
After some considerations, I've decided to go this way for all formats in https://github.com/WeblateOrg/weblate/pull/9584.